packages/x-components/src/x-modules/empathize/components/empathize.vue

```diff
@@ -56,14 +56,7 @@
      *
      * @public
      */
-    @Prop({
-      default: () => [
-        'UserClosedEmpathize',
-        'UserSelectedASuggestion',
-        'UserPressedEnter',
-        'UserBlurredSearchBox'
-      ]
-    })
+    @Prop({ default: () => ['UserClosedEmpathize', 'UserSelectedASuggestion', 'UserPressedEnterKey', 'UserBlurredSearchBox'] })
```

Comment:
Add the type here to prevent adding wrong values.

```suggestion
    @Prop({ default: (): XEvent[] => ['UserClosedEmpathize', 'UserSelectedASuggestion', 'UserPressedEnterKey', 'UserBlurredSearchBox'] })
```